### PR TITLE
Fixes typo in problem explanation of WebP Express

### PR DESCRIPTION
**Pull request by mufucaw**
Closes #

## Summary

**[WordPress Plugins and Themes Known Issues](https://docs.pantheon.io/plugins-known-issues)** - Fixes typo in the WebP Express problem description.

## Effect

The following changes are already committed:

*Changes "write-only" to "read-only"

## Remaining Work and Prerequisites

The following changes still need to be completed:

None

### Dependencies and Timing

None

**Release**:
- [x] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Review by CdrMarks — APPROVED** [review]
Thanks for the PR.  The Docs team are the CODEOWNERS.  They'll review and merge this when approved.

**Review by michellecolon-pantheon — APPROVED** [review]
LGTM

**Pull request merged**
